packages/twenty-front/src/modules/workflow/workflow-steps/components/WorkflowRunStepOutputDetail.tsx

```diff
@@ -30,24 +34,38 @@ export const WorkflowRunStepOutputDetail = ({ stepId }: { stepId: string }) => {
     return null;
   }
 
-  const stepOutput = workflowRun.output.stepsOutput[stepId];
+  const stepOutput = workflowRun.output.stepsOutput[stepId] as
+    | WorkflowExecutorOutput
+    | undefined;
 
   const stepDefinition = getStepDefinitionOrThrow({
     stepId,
     trigger: workflowRun.output.flow.trigger,
     steps: workflowRun.output.flow.steps,
   });
-  if (stepDefinition?.type !== 'action') {
-    throw new Error('The output tab must be rendered with an action step.');
+  if (
+    !isDefined(stepDefinition?.definition) ||
+    !isDefined(stepDefinition.definition.name)
+  ) {
+    throw new Error('The step is expected to be properly shaped.');
```

Comment:
We now support rendering triggers in this component. It involved refactoring a few things.